### ISSUE TITLE
increased lineHeight cuts off multi-line cells

### ISSUE THREAD

**Issue by jtarvainen** [simonbengtsson/jsPDF-AutoTable#767]
I found the following bug in jsPDF-AutoTable v3.5.13:

Using a line height greater than the default (1.15) is not taken into account by jsPDF-AutoTable, causing multi-line cells to get cut off vertically.

Steps to reproduce:

```
const doc = new jsPDF({
    lineHeight: 1.5
})

doc.autoTable({
    body: [
        ['text \n on \n four \n lines'],
        ['text on one line'],
        ['text on one line'],
    ],
})
```

This issue was [previously pointed on in August 2020 by another user](https://github.com/simonbengtsson/jsPDF-AutoTable/issues/712), but for some reason that issue was closed, even though this is a clearly a bug.

**Comment by simonbengtsson** [comment]
True! There is a stale PR open for this https://github.com/simonbengtsson/jsPDF-AutoTable/pull/754, but a new PR would be great.

**Comment by mmghv** [comment]
Released in [v3.7.0](https://github.com/simonbengtsson/jsPDF-AutoTable/releases/tag/v3.7.0)
